### ISSUE TITLE
The .txt file

### ISSUE THREAD

**Issue by Johnny00520** [YifanLi9/Final-Project-YifanLi#1]
When I tried to run your program, it gives me the seg-fault. And the reason is your name of .txt file has a space between words. I changed CourseList_copy.txt, and it wouldn't give me seg-fault anymore. Or you can write getline function in your commend line argument in main function.

**Comment by YifanLi9** [comment]
Thank you for your reply. After you changed it, can you run the program?

**Comment by Johnny00520** [comment]
Yes, it does, but your option 3, which is quit. It doesn't really quit

**Comment by imcmorran** [comment]
Also, make sure to update your readme file to include instructions for using a command line argument. It would be helpful to the TA to know this without having to read through the code. Never assume anything.
